tests/TableViewTest.cc

```diff
@@ -23,7 +23,9 @@
 #include <future>
 
 #include "HttpHelper.h"
+#include "LogUtils.h"
 #include "PulsarFriend.h"
+#include "TopicName.h"
```

Comment:
Is this redundant?

Reply:
No. This file uses `TopicDomain` in `TopicName.h`, which was included from another header before this patch.